src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -184,4 +183,106 @@ public void Chat_007_banUser() {
     chatUserStaff.banUser(userToBeBanned);
     chatUserStaff.unBanUser(userToBeBanned);
   }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_008", "Chat", "Modals", "ChatTests"})
+  public void Chat_008_userBlockedCanNotCreatePrivateMessage() {
+    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+
+    switchToWindow(0);
+    chatUserOne.clickOnDifferentUser(userFive);
+    chatUserOne.selectPrivateMessageToUser(userFive);
+    chatUserOne.blockPrivateMessageFromUser(userFive);
+
+    switchToWindow(1);
+    chatWindow.refreshPage();
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.verifyBlockedUserIsUnableWritePrivateMessage(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_009", "Chat", "Modals", "ChatTests"})
+  public void Chat_009_userCanOpenMessageWall() throws InterruptedException {
+    openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.clickOpenUserMessageWall(userOne);
+    chatUserFive.switchToSecondTab();
+    chatWindow.refreshPage();
+    chatUserFive.verifyMessageWallOpened(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_010", "Chat", "Modals", "ChatTests"})
+  public void Chat_010_userCanOpenContributions() throws InterruptedException {
+    openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.clickOpenUserContributions(userOne);
+    chatUserFive.switchToSecondTab();
+    chatWindow.refreshPage();
+    chatUserFive.verifyUserContributionsOpened(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_011", "Chat", "Modals", "ChatTests"})
+  public void Chat_011_userCanNotBlockPrivateMessages() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)